lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -315,8 +315,9 @@ def stop_job( self, job ):
             if kill_script is None:
                 self.ds.kill( ext_id )
             else:
-                # FIXME: hardcoded path
-                subprocess.Popen( [ '/usr/bin/sudo', '-E', kill_script, str( ext_id ), str( self.userid ) ], shell=False )
+                command = kill_script.split()
```

Comment:
Here it would also be better to use `shlex.split(kill_script)`, I should have mentioned it in the previous review round.